README.md

```diff
@@ -22,14 +26,33 @@ not to show a table listing all the found file extensions and their respective
 frequencies, so that it will only display the total number of files.
 
 By default, it will ignore files and directories that are supposed to be
-hidden (with names starting with '.', but you can add the `-a` or `--all` optional
+hidden, but you can add the `-a` or `--all` optional
 switch argument to make it count all files.
 
+Hidden files and directories
+
+Windows: files and directories for which FILE_ATTRIBUTE_HIDDEN is true
+
+Linux, Mac OS: those with names starting with "."(dot)
+
 This utility can also be used to search for files that have a certain file extension
 (using `-fe` or `--file-extension`) and, optionally, display a short preview (`-p`or 
 `--preview`) for text files. The size of the preview text sample can optionally be
 customized by using the `-ps` or `--preview-size` argument followed by an integer number.
 
+The list of file types for which preview is available can be viewed with the `-st` or `--supported-types` argument.
+The names of extensions are case sensitive. The results for `ini` and `INI` will be different.
```

Comment:
I am thinking that we should make it case-insensitive by default, and add a new option (`-cs`or `--case-sensitive` ) to search or count with case-sensitiveness. It shouldn't be very complicated, right?

Reply:
#50